components/match2/wikis/leagueoflegends/big_match.lua

```diff
@@ -0,0 +1,904 @@
+							"Cosmic Insight",
+						},
+						["tree"] = "Inspiration",
+					},
+				},
+				["spells"] = {
+					"Flash",
+					"Teleport",
+				},
+				["trinket"] = "Oracle Lens",
+				["wardsPlaced"] = 10,
+			},
+			{
+				["assists"] = 2,
+				["champion"] = "Aphelios",
+				["creepScore"] = 209,
+				["damageDone"] = 94416,
+				["deaths"] = 5,
+				["gold"] = 7583,
+				["id"] = "dresscode",
+				["items"] = {
+					"Bloodthirster",
+					"Berserker's Greaves",
+					"Noonquiver",
+					"Control Ward",
+					"Pickaxe",
+				},
+				["kills"] = 0,
+				["role"] = "bottom",
+				["runeData"] = {
+					["primary"] = {
+						["runes"] = {
+							"Magical Footwear",
+							"Biscuit Delivery",
+							"First Strike",
+							"Cosmic Insight",
+						},
+						["tree"] = "Inspiration",
+					},
+					["secondary"] = {
+						["runes"] = {
+							"Manaflow Band",
+							"Scorch",
+						},
+						["tree"] = "Sorcery",
+					},
+				},
+				["spells"] = {
+					"Flash",
+					"Heal",
+				},
+				["trinket"] = "Farsight Alteration",
+				["wardsPlaced"] = 9,
+			},
+			{
+				["assists"] = 3,
+				["champion"] = "Ahri",
+				["creepScore"] = 203,
+				["damageDone"] = 108743,
+				["deaths"] = 1,
+				["gold"] = 7806,
+				["id"] = "Ace",
+				["items"] = {
+					"Doran's Ring",
+					"Everfrost",
+					"Broken Stopwatch",
+					"Ionian Boots of Lucidity",
+					"Horizon Focus",
+				},
+				["kills"] = 1,
+				["role"] = "middle",
+				["runeData"] = {
+					["primary"] = {
+						["runes"] = {
+							"Cut Down",
+							"Presence of Mind",
+							"Conqueror",
+							"Legend: Alacrity",
+						},
+						["tree"] = "Precision",
+					},
+					["secondary"] = {
+						["runes"] = {
+							"Manaflow Band",
+							"Scorch",
+						},
+						["tree"] = "Sorcery",
+					},
+				},
+				["spells"] = {
+					"Flash",
+					"Teleport",
+				},
+				["trinket"] = "Oracle Lens",
+				["wardsPlaced"] = 4,
+			},
+		},
+		["towerKills"] = 0,
+	},
+	["team2Score"] = 0,
+	["winner"] = 1,
+}
+end
+
+local KEYSTONES = Table.map({
+	-- Precision
+	'Press the Attack',
+	'Lethal Tempo',
+	'Fleet Footwork',
+	'Conqueror',
+
+	-- Domination
+	'Electrocute',
+	'Predator',
+	'Dark Harvest',
+	'Hail of Blades',
+
+	-- Sorcery
+	'Summon Aery',
+	'Arcane Comet',
+	'Phase Rush',
+
+	-- Resolve
+	'Grasp of the Undying',
+	'Aftershock',
+	'Guardian',
+
+	-- Inspiration
+	'Glacial Augment',
+	'Unsealed Spellbook',
+	'First Strike',
+}, function (_, value)
+	return value, true
+end)
+
+local ROLE_ORDER = Table.map({
+	'top',
+	'jungle',
+	'middle',
+	'bottom',
+	'support',
+}, function (key, value)
+	return value, key
+end)
+
+local NOT_PLAYED = 'np'
+local DEFAULT_ITEM = 'EmptyIcon'
+local TEAMS = Array.range(1, 2)
+
+function BigMatch.run(frame)
+	local args = Arguments.getArgs(frame)
+
+	---@type BigMatch
+	local bigMatch = BigMatch()
+
+	args = bigMatch:_contextualEnrichment(args)
+
+	local match = bigMatch:_match2Director(args)
+
+	local renderModel = match
```

Comment:
isn't match and renderModel the same in this case?
like one just aliasing

maybe Table.copy?